drp.tex

```diff
@@ -54,9 +54,8 @@ \subsection{Calibration}
 %LPG: Great, thank you
 \subsubsection{PSF Modeling}
 \label{ssec:psf_modelling}
-\gls{PSF} modeling in \gls{DP1} uses the Piff \citep{DES:2020vau} algorithm.
-Piff models represent the \gls{PSF} on a pixel-by-pixel basis and interpolate its parameters across a single CCD using two-dimensional polynomials.
-Piff utilizes its Pixel grid model with a fourth-order polynomial interpolation per \gls{CCD}, except in the u-band, where star counts are insufficient to support a fourth-order fit.
+\gls{PSF} modeling in \gls{DP1} uses the Piff \citep{DES:2020vau} package.
+Our configuration of Piff utilizes its \texttt{PixelGrid} model with a fourth-order polynomial interpolation per \gls{CCD}, except in the u-band, where star counts are insufficient to support a fourth-order fit.
```

Comment:
This (modified) sentence and the one below said the same thing, and it's important to clarify that this is how we're running Piff, not something Piff imposed on us.